compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/ZipUtils.java

```diff
@@ -59,38 +58,52 @@ static void generateBalo(BLangPackage bLangPackage, String projectPath, Stream<P
             try {
                 Files.createDirectories(destPath);
             } catch (IOException e) {
-                throw new BLangCompilerException("Error occurred when creating directories in " +
-                                                         "./ballerina/repo/ to save the generated balo");
+                throw new BLangCompilerException("error creating directories in ./ballerina/repo/ to save the " +
+                                                         "generated balo");
```

Comment:
shouldn't say balo to user.

Reply:
Changed balo to artifact : aee8b8a68a74934a7076d0ebd12ba1592eeadd16